osu.Game/Screens/Play/ReplayPlayer.cs

```diff
@@ -17,7 +17,7 @@ public ReplayPlayer(Score score)
         protected override void LoadComplete()
         {
             base.LoadComplete();
-            RulesetContainer.SetReplayScore(score);
+            RulesetContainer?.SetReplayScore(score);
```

Comment:
I'd argue that this shouldn't ever be the case in `ReplayPlayer`'s `LoadComplete`. When does this occur?

Reply:
Check the place it's initialised